main_app/templates/main_app/home.html

```diff
@@ -143,6 +143,68 @@
         transform: rotate(360deg);
```

Comment:
Because of the changes in the `home.html` page, the gif image is going inside the nav. Fix it.
![image](https://user-images.githubusercontent.com/53635792/112320436-041f1280-8cd5-11eb-82be-25213766dbeb.png)